committing.rst

```diff
@@ -246,81 +247,70 @@ issue's page in the tracker, for each mentioned issue.
-  of the person working on a patch.  If you are using the rebase_ extension,
-  consider adding the ``--collapse`` option to ``hg rebase``.  The collapse_
-  extension is another choice.
+  of the person working on a patch.
 
-Because of these constraints, it can be practical to use other approaches
-such as mq_ (Mercurial Queues), in order to maintain patches in a single
-local repository and to push them seamlessly when they are ready.
+It recommended to keep a fork of the main repository around, as it allows simple
+reversion of all local changes (even "committed" ones) if your local clone gets
+into a state you aren't happy with.
 
-It can also be useful to keep a pristine clone of the main repository around,
-as it allows simple reversion of all local changes (even "committed" ones) if
-your local clone gets into a state you aren't happy with.
 
-
-.. _Mercurial: http://www.hg-scm.org/
-.. _mq: https://www.mercurial-scm.org/wiki/MqExtension
-.. _rebase: https://www.mercurial-scm.org/wiki/RebaseExtension
-.. _collapse: https://www.mercurial-scm.org/wiki/CollapseExtension
+.. _Git: https://git-scm.com/
 
 
 Minimal Configuration
 ---------------------
 
-If you use Mercurial as a committer of patches (your own or others), you should
+If you use Git as a committer of patches (your own or others), you should
 set up some basic options in your `configuration file`_.
 
-* On *UNIX* (including Mac OS X), you must edit ``~/.hgrc``.
+* On *UNIX* (including Mac OS X), you must edit ``~/.gitconfig``.
+// TODO find out how windows handles git
 * Under *Windows*, use TortoiseHg's graphical settings dialog to make changes
   to ``mercurial.ini``. Alternatively, you may edit the file directly at
   ``%USERPROFILE%\mercurial.ini``.
 
 
 Here are the minimal options you need to activate:
 
-* your *username*: this setting defines the name that will be used when you
-  :ref:`commit <hg-commit>` changes.  The usual convention is to also include
-  an e-mail contact address in there::
-
-   [ui]
-   username = Your Name <email@example.org>
+* your *name* and *email*: these settings defines what will be used when you
+  :ref:`commit <git-commit>` changes::
 
-* *extended diffing*: this setting enables an `extended diff format`_
-  which is more useful than the standard unified diff format as it includes
-  metadata about file copies, permission bits, and is able to represent
-  binary files::
+   [user]
+   name = Your Name
+   email = email@example.org
 
-   [diff]
-   git = on
-
-* *Under Windows*, you should also enable the `eol extension`_, which will
+* *Under Windows*, you should also enable the *autocrlf* option, which will
   fix any Windows-specific line endings your text editor might insert when you
   create or modify versioned files.  The public repository has a hook which
   will reject all changesets having the wrong line endings, so enabling this
   extension on your local computer is in your best interest.
 
+   [core]
+   autocrlf = true
+
```

Comment:
Are there any other settings that come to your mind or should I ask at core-workflow list?